tools/render-test/render-d3d11.cpp

```diff
@@ -612,165 +578,149 @@ void* D3D11Renderer::map(ID3D11Buffer * buffer, MapFlavor flavor)
     // We update our constant buffer per-frame, just for the purposes
     // of the example, but we don't actually load different data
     // per-frame (we always use an identity projection).
-    D3D11_MAPPED_SUBRESOURCE dxMapped;
-    HRESULT hr = dxContext->Map(dxBuffer, 0, dxMapFlavor, 0, &dxMapped);
+    D3D11_MAPPED_SUBRESOURCE mappedSub;
+    HRESULT hr = m_immediateContext->Map(buffer, 0, mapType, 0, &mappedSub);
     if (FAILED(hr))
         return nullptr;
 
-    return dxMapped.pData;
+    return mappedSub.pData;
 }
 
 void* D3D11Renderer::map(Buffer* buffer, MapFlavor flavor)
 {
     return map(((BufferImpl*)buffer)->buffer, flavor);
 }
 
-void D3D11Renderer::unmap(ID3D11Buffer * buffer)
+void D3D11Renderer::unmap(ID3D11Buffer* buffer)
 {
-    auto dxContext = dxImmediateContext;
-    dxContext->Unmap(buffer, 0);
+    m_immediateContext->Unmap(buffer, 0);
 }
 
-void D3D11Renderer::unmap(Buffer* buffer)
+void D3D11Renderer::unmap(Buffer* bufferIn)
 {
-    auto dxBuffer = ((BufferImpl*)buffer)->buffer;
-    unmap(dxBuffer);
+    unmap(((BufferImpl*)bufferIn)->buffer);
 }
 
-void D3D11Renderer::setInputLayout(InputLayout* inputLayout)
+void D3D11Renderer::setInputLayout(InputLayout* inputLayoutIn)
 {
-    auto dxContext = dxImmediateContext;
-    auto dxInputLayout = (ID3D11InputLayout*)inputLayout;
+    auto inputLayout = (ID3D11InputLayout*)inputLayoutIn;
 
-    dxContext->IASetInputLayout(dxInputLayout);
+    m_immediateContext->IASetInputLayout(inputLayout);
 }
 
 void D3D11Renderer::setPrimitiveTopology(PrimitiveTopology topology)
 {
-    auto dxContext = dxImmediateContext;
-
-    D3D11_PRIMITIVE_TOPOLOGY dxTopology;
+    D3D11_PRIMITIVE_TOPOLOGY primTopology;
     switch (topology)
     {
         case PrimitiveTopology::TriangleList:
-            dxTopology = D3D11_PRIMITIVE_TOPOLOGY_TRIANGLELIST;
+            primTopology = D3D11_PRIMITIVE_TOPOLOGY_TRIANGLELIST;
             break;
 
         default:
             return;
     }
 
-    dxContext->IASetPrimitiveTopology(dxTopology);
+    m_immediateContext->IASetPrimitiveTopology(primTopology);
 }
 
-void D3D11Renderer::setVertexBuffers(UInt startSlot, UInt slotCount, Buffer* const* buffers, UInt const* strides, UInt const* offsets)
+void D3D11Renderer::setVertexBuffers(UInt startSlot, UInt slotCount, Buffer*const* buffersIn, const UInt* stridesIn, const UInt* offsetsIn)
```

Comment:
Hmm... I think I get it now. The parameter names got the `In` suffix not to clarify their role to the caller (who might be looking at these declarations using Intellisense) but to avoid conflict with the local variables that now get the same name.

Yeah, I'm not convinced that dropping the `dx` prefix was a net win here.